scripts/test.js

```diff
@@ -8,65 +8,59 @@
 
 'use strict';
 
-var configFile;
-var networkFile;
 var path = require('path');
 var fs = require('fs-extra');
 var exec = require('child_process').exec;
-
-function setConfig(file) {
-    configFile = path.join('../..', file);
-}
-
-function setNetwork(file) {
-    networkFile = path.join('../..', file);
-}
+const Util = require('../src/comm/util');
 
 function main() {
-    if(process.argv.length < 3) {
-        console.log('undefined benchmark name, should be "npm test -- benchmark-name [options]"');
-        process.exit(1);
-    }
-
-    let benchmark = process.argv[2];
-    let runDir = path.join(__dirname, '../benchmark', benchmark);
-    if(!fs.existsSync(runDir)) {
-        console.log('directory ' + runDir + ' does not exist');
-        process.exit(1);
-    }
-
-    let runExe = path.join(runDir, 'main.js');
-    if(!fs.existsSync(runExe)) {
-        console.log('file ' + runExe + ' does not exist');
-        process.exit(1);
-    }
-
     let program = require('commander');
     program
-        .option('-c, --config <file>', 'config file of the benchmark', setConfig)
-        .option('-n, --network <file>', 'config file of the blockchain system under test', setNetwork)
+        .option('-c, --config <file>', 'config file of the benchmark')
+        .option('-n, --network <file>', 'config file of the blockchain system under test')
         .parse(process.argv);
-    let cmd = 'node main.js';
-    if(typeof configFile === 'string') {
-        cmd += ' -c ';
-        cmd += configFile;
+
+    let logger = Util.getLogger('scripts/test.js');
+    let absConfigFile;
+    if(typeof program.config === 'undefined') {
+      logger.error('config file is required');
+      return;
```

Comment:
Since this is the entry point of the process, an error code could be returned

Reply:
That's the idea, so we can't run without the config set